src/assets/js/alma-admin.js

```diff
@@ -223,3 +223,20 @@ function AlmaAdminFeePlan(helper ) {
         showPlan: showPlan
     }
 }
+
+var coll = jQuery(".alma-legal-checkout-collapsible");
+var chevron = jQuery('#alma-legal-collapse-chevron');
+var i;
+for (i = 0; i < coll.length; i++) {
```

Comment:
remove loop here and get `collapsible` element by id (and rename `coll` to `collapsible`

Reply:
Done